Branch-SDK/src/main/java/io/branch/referral/Defines.java

```diff
@@ -100,9 +98,7 @@
         DeepLinkPath(Branch.DEEPLINK_PATH),
         
         AndroidAppLinkURL("android_app_link_url"),
-        AndroidPushNotificationKey("branch"),
```

Comment:
I'm not opposed to these, but this will cause a major version bump.  Let's make sure we get sign off from PM.

Reply:
@csalmi-branch @derrickstaten, these were basically publicly accessible (but NOT advertised) strings that we use as keys in an Android intent. They were incorrectly classified as `JsonKeys` instead of `IntentKeys`. Is it OK to go ahead with this change and make them `IntentKeys`? (if any of the users utilized these, then upon updating to the new SDK version, their builds would break at compile time)

Reply:
Is there any way to not break our users' code, if they were utilizing this? I'm generally opposed to making changes that cause users' code to break.

If not, please proceed.